app/mailers/user_mailer.rb

```diff
@@ -1,20 +1,35 @@
 # frozen_string_literal: true
 
 class UserMailer < Devise::Mailer
+  DEVISE_ACTIONS = [
```

Comment:
This was the least-bad thing I could think of to highlight the fact that the devise-overriding actions aren't getting the same changes as the others.

I think there's room for future refactor here to keep all the devise ones in their own mailer and move the rest to other mailer(s), but didn't want this PR to get even bigger.